### PR TITLE
Add speech recognition and synthesis features

### PR DESCRIPTION
**Pull request by foolip**

### CODE REVIEW

**Comment by foolip** [comment]
Ping @ddbeck

File: feature-group-definitions/speech-recognition.yml [hunk 1]
```diff
@@ -0,0 +1,4 @@
+name: Speech Recognition
```
Comment (ddbeck) [inline feature-group-definitions/speech-recognition.yml:1]:
```suggestion
name: Speech recognition
```

File: feature-group-definitions/speech-synthesis.yml [hunk 2]
```diff
@@ -0,0 +1,4 @@
+name: Speech Synthesis
```
Comment (ddbeck) [inline feature-group-definitions/speech-synthesis.yml:1]:
```suggestion
name: Speech synthesis
```

**Review by ddbeck — APPROVED** [review]
A couple nits, but otherwise good.

**Pull request merged**